### PR TITLE
feat: reorder migration skeleton and create new section

### PR DESCRIPTION
**Pull request by fzn0x**
Takeover https://github.com/sequelize/website/issues/94#issuecomment-1100644109

### CODE REVIEW

**Comment by fzn0x** [comment]
Build catch an unknown error, should be checked on log.tmp.txt

```sh
Run netlify deploy --dir build --message '' > log.tmp.txt 2>&1
Error: Process completed with exit code 1.
```

File: docs/other-topics/migrations.md [hunk 1]
```diff
@@ -445,57 +453,57 @@ The configuration file is by default a JSON file called `config.json`. But somet
 Thankfully, the Sequelize CLI can read from both `.json` and `.js` files. This can be setup with `.sequelizerc` file. You just have to provide the path to your `.js` file as the `config` option of your exported object:
 
 ```js
-const path = require('path');
+const path = require("path");
```
Comment (ephys) [inline docs/other-topics/migrations.md:456]: Don't change the code style of the document (this is valid for all code style related changes that occurred)

**Review by ephys — CHANGES_REQUESTED** [review]

File: docs/other-topics/migrations.md (hunk 1 above)
Comment (fzn0x) [reply docs/other-topics/migrations.md:456]: > Don't change the code style of the document (this is valid for all code style related changes that occurred)

I just turn off my prettier extension in my vscode, it seems still working for markdown :/ I will revert

Comment (fzn0x) [reply docs/other-topics/migrations.md]: > Don't change the code style of the document (this is valid for all code style related changes that occurred)

My prettier extension got messed up, let me fix it

**Review by ephys — CHANGES_REQUESTED** [review]
Moving that section that high up in the documentation doesn't make sense to me. It should at the very least be after "Creating the first Model (and Migration)"

File: docs/other-topics/migrations.md [hunk 2]
```diff
@@ -39,6 +39,179 @@ This will create following folders
 - `migrations`, contains all migration files
 - `seeders`, contains all seed files
 
+## Initiating migration files
```
Comment (ephys) [inline docs/other-topics/migrations.md:42]: I suggest

```suggestion
## Writing a migration
```

**Review by ephys — CHANGES_REQUESTED** [review]

File: docs/other-topics/migrations.md (hunk 2 above)
Comment (fzn0x) [reply docs/other-topics/migrations.md:42]: Much better

**Comment by fzn0x** [comment]
> Moving that section that high up in the documentation doesn't make sense to me. It should at the very least be after "Creating the first Model (and Migration)"

It should be after project bootstrapping, what I do before using the migrations CLI is

1. Installing the CLI
2. Project bootstrapping
3. Initiating the migration

If you follow the steps above as first reader, it's actually made development easier with the guide, further steps like sequelizerc must be optional and not required.

It's actually not sensible but much beginner-friendly

**Comment by fzn0x** [comment]
Creating the first Model (and Migration) is much like the advanced version one, So I put it under, but that's just my opinion.

**Comment by fzn0x** [comment]
Also, you can create models without model:generate too :)

**Comment by ephys** [comment]
I disagree. It doesn't make sense to create a migration before you have anything to migrate. It makes even less sense to start writing migrations before the barebones of the project is installed, including the configuration

**Comment by fzn0x** [comment]
> I disagree. It doesn't make sense to create a migration before you have anything to migrate. Creating models is the logical first step when you start a project. Migrations that modify models will always come later than that.
> 
> It makes even less sense to start writing migrations before the barebones of the project is installed, including the configuration

What do you think @WikiRik , is it above Creating models and migration with model:generate or below?

**Comment by fzn0x** [comment]
Maybe it will easier to create decision when there is more than 2 people on it

**Comment by WikiRik** [comment]
I'll take a look at this later today

**Comment by WikiRik** [comment]
I think this page might need more work than this. I think we should split this into basic usage and then more advanced usage with the various examples.

For this PR I would say that we could move the skeleton, generation command and first example to come between 'Creating the first Model (and Migration)' and 'Running Migrations'. There we can also reference the other examples that are then in the current location later in the page.

**Comment by ephys** [comment]
I have started a sort of rewrite of a lot of the documentation, I haven't gotten to migrations yet but I can certainly include more changes when I get to it

If you have ideas, I've opened https://github.com/sequelize/website/issues/93 to collect things that need to be included in the rewrite

**Comment by WikiRik** [comment]
> This PR would already be a good improvement without rewriting the whole page imo

Agree. What will we do with the location of this block? Move it so it is between 'Creating the first Model (and Migration)' and 'Running Migrations'?

**Comment by fzn0x** [comment]
I will re-create the PR inside sequelize-website repo. Just to make sure to pass the build.

**Pull request closed without merging**